Assets/MRTK/Services/InputSimulation/SimulatedHandDataProvider.cs

```diff
@@ -293,19 +307,23 @@ private void SimulateUserInput(MouseDelta mouseDelta)
                 {
                     isSimulatingRight = false;
                 }
+                if(isSimulatingGaze)
+                    lastSimulationGaze = time;
             }
 
             mouseRotation.Update(profile.HandRotateButton, cancelRotationKey, false);
 
             SimulateHandInput(ref lastHandTrackedTimestampLeft, HandStateLeft, isSimulatingLeft, IsAlwaysVisibleLeft, mouseDelta, mouseRotation.IsRotating);
             SimulateHandInput(ref lastHandTrackedTimestampRight, HandStateRight, isSimulatingRight, IsAlwaysVisibleRight, mouseDelta, mouseRotation.IsRotating);
+            SimulateHandInput(ref lastHandTrackedTimestampGaze, HandStateGaze, isSimulatingGaze, false, mouseDelta, mouseRotation.IsRotating);
```

Comment:
Since this is a new (and exciting!) behavior that you can left click to select items, please document it in the [input simulation documentation](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/InputSimulation/InputSimulationService.html) as well as the 2.4 release notes. A GIF for the release notes would be nice.